Versions in install_if_not_exists

If you check out the main branch at the latest commit (`7ce0081`):

```bash
git checkout main
cd examples/morpheusvm
./scripts/lint.sh
```

You'll see the following output:

```bash
Running 'golangci_lint gci' at: Fri Aug 30 10:49:17 AM UTC 2024
START: 'golangci_lint' at Fri Aug 30 10:49:17 AM UTC 2024
WARN [config_reader] The configuration option `run.skip-files` is deprecated, please use `issues.exclude-files`. 
WARN The linter 'interfacer' is deprecated (since v1.38.0) due to: The repository of the linter has been archived by the owner.  
WARN The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
ERRO [linters_context] interfacer: This linter is fully inactivated: it will not produce any reports. 
ERRO [linters_context] structcheck: This linter is fully inactivated: it will not produce any reports. 
cmd/morpheus-cli/cmd/chain.go:40:31: unused-parameter: parameter 'args' seems to be unused, consider removing or renaming it as _ (revive)
        RunE: func(_ *cobra.Command, args []string) error {
                                     ^
cmd/morpheus-cli/cmd/prometheus.go:20:31: unused-parameter: parameter 'args' seems to be unused, consider removing or renaming it as _ (revive)
        RunE: func(_ *cobra.Command, args []string) error {
                                     ^
FAIL: 'golangci_lint' failed at Fri Aug 30 10:49:18 AM UTC 2024
```

However, if you check the `morpheusvm-lint` CI action logs for the same commit [here](https://github.com/ava-labs/hypersdk/actions/runs/10630947896/job/29471013040), everything passes:

```
SUCCESS: 'gci' completed...
ALL SUCCESS!
```

This inconsistency suggests a possible issue with running the script from the wrong folder, a script mismatch, or a version mismatch.

--- a/scripts/lint.sh
+++ b/scripts/lint.sh
@@ -35,7 +35,7 @@ TESTS=${TESTS:-"golangci_lint gci"}
 
 # https://github.com/golangci/golangci-lint/releases
 function test_golangci_lint {
-  install_if_not_exists golangci-lint github.com/golangci/golangci-lint/cmd/golangci-lint@"$GOLANGCI_LINT_VERSION"
+  go install -v github.com/golangci/golangci-lint/cmd/golangci-lint@"$GOLANGCI_LINT_VERSION"
 
   golangci-lint run --config .golangci.yml
 }

